src/metadata_tool/sources/confluence.clj

```diff
@@ -15,10 +15,17 @@
 ; limitations under the License.
 ;
 (ns metadata-tool.sources.confluence
-  (:require [clojure.string        :as str]
-            [clj-http.client       :as http]
-            [clj-http.conn-mgr     :as conn]
-            [metadata-tool.config  :as cfg]))
+  (:require [clojure.string                :as str]
+            [clj-http.client               :as http]
+            [clj-http.conn-mgr             :as conn]
+            [metadata-tool.tools.selenium  :as selenium]
+            [metadata-tool.config          :as cfg])
+  (:import
+   (java.time LocalTime)
```

Comment:
I don't see this used, but if it was, consider `clj-time`.  Very nice idiomatic clojure wrapper around joda-time

Reply:
You're right, it's not used, I "manually" parse the date in `parse-date` function; I think we already use `joda`, pretty cool indeed. Reason we added `LocalTime` was because it provides a `parseDate` method to validate a date, probably joda does too.

I'll get rid of this block, TY.